src/vendor/utils.js

```diff
@@ -1,9 +1,22 @@
 /* global window */
 /* eslint-disable no-restricted-syntax */
 
+const { isArray } = Array;
+const zero = () => 0;
+
+function array(length = 0) {
```

Comment:
Narrative explanation of how `new Array()` is broken would be helpful in a comment here.

Reply:
Each source file has a test of the same name. Here is the reason `array()` should be used instead of `new Array()`:

https://github.com/mozilla-frontend-infra/firefox-health-dashboard/blob/056032f7f8385a5f02246e8263f81d02414e0686/test/vendor/utils.test.js#L63

If you mean for others to better understand what this code does: Yes, I will add a comment.